portfolio/src/main/webapp/style.css

```diff
@@ -1,5 +1,16 @@
 /* General Styles */
 
+body {
+    /* background: url("https://www.toptal.com/designers/subtlepatterns/patterns/y-so-serious-white.png"); */
```

Comment:
You commented out some styles here and below. Did you mean to remove them?

Reply:
Fixed it.